react/features/invite/components/LockState.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+
+import { translate } from '../../base/i18n';
+
+/**
+ * A React Component for displaying the conference lock state.
+ */
+class LockState extends Component {
```

Comment:
I realise that we don't have a naming convention on Components or maybe we have and I don't know about it, but I would suggest that we add some word at the end that indicates what kind of component is rendered in the file. For example AddPassword.js will become AddPasswordContainer or AddPasswordForm and RemovePassword.js will become RemovePasswordContainer. PasswordOverview.js will become PasswordContainer probably and LockState will be LockStatePanel or LockStateLabel.